features/put_filter_outputs.feature

```diff
@@ -87,7 +91,7 @@ Feature: Put Filter Outputs Private Endpoints Not Enabled
     ]
     """
 
-  Scenario: PUT filter outputs successfully with existing ID in DB
+Scenario: PUT filter outputs successfully with existing ID in DB
```

Comment:
The tests look good, perhaps it would be good to involve a test in the event that mongo is down?

Reply:
There is scenario below "Put a filter output with broken mongo db" that covers this